api/src/main/java/javax/enterprise/event/FireAsyncException.java

```diff
@@ -0,0 +1,34 @@
+package javax.enterprise.event;
+
+/**
+ * <p>
+ * This Exception list all exceptions that occurred during notification of async observers of an event.
```

Comment:
If this exception is relevant to exceptions thrown out of async observers only, we should define what happens with exceptions raised by sync observers.